gretl/src/main/java/ch/so/agi/gretl/tasks/Publisher.java

```diff
@@ -81,6 +81,11 @@ public String getDataIdent() {
     public Endpoint getTarget() {
         return target;
     }
+
+    /*
+     *  @InputFile kann hier nicht verwendet werden, da die Datei existieren muss.
+     *  Bei einem ersten Run dieses Tasks kann es sein, dass die Datei noch nicht existiert.
+     */
     @Input
```

Comment:
Hier wäre @OutputFile nicht logisch aus meiner Sicht.  Auch @InputFile passt irgendwie nicht. In der TaskAction wird mit dem Pfad dann eine Datei erstellt/geholt. Daher habe ich @Input gelassen.

Reply:
@edigonzales